justfile

```diff
@@ -95,19 +95,3 @@ generate-drawing-examples-montage:
         {{doc_assets_dir}}/display*.png \
         -tile 6x2 -background none -geometry 128x128+4+4 miff:- | \
     convert - -trim {{doc_assets_dir}}/all_drawing_ops.png
-
-# Generates a screenshot of an example
-generate-example-screenshot example:
-    @mkdir -p "{{screenshots_dir}}"
-    # Generating {{example}} screenshot
-    EG_SIMULATOR_DUMP="{{screenshots_dir}}/{{example}}.png" \
```

Comment:
The `screenshots_dir` declaration at the top could be removed as well.

Reply:
Good catch. I've also removed the unused `ci_build_image` variable.